tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -215,3 +215,15 @@ func tryRandomAction(ctx context.Context, t *testing.T, opts engine.ActionOpts)
 
 	return err
 }
+
+// tryExecAction runs the given engine action with options and masks no-op errors.
+func tryExecAction(ctx context.Context, t *testing.T, action engine.ActionKey, actionOpts map[string]string) error { //nolint:thelper
+	_, err := eng.ExecAction(ctx, action, actionOpts)
+	// Ignore the dir-not-found error, wrapped as no-op error.
+	if errors.Is(err, robustness.ErrNoOp) {
+		t.Log("Delete action resulted in no-op")
+		return nil
+	}
+
+	return err
```

Comment:
What about moving the `require.NoError` check to the helper and removing the returned `error` from the function signature?

Note: it is OK to use "testing" patterns in the robustness framework.

Reply:
done